src/platforms/wayland/platform_symbols.cpp

```diff
@@ -64,7 +64,7 @@ mg::PlatformPriority probe_graphics_platform(
 {
     mir::assert_entry_point_signature<mg::PlatformProbe>(&probe_graphics_platform);
 
-    return mpw::connection(options) ?
+    return mpw::supported(options) ?
         mg::PlatformPriority::best :
         mg::PlatformPriority::unsupported;
```

Comment:
The thought occurs that you *could* call `wl_display_connect(NULL)` and check for successful connection and return `::supported`.

Reply:
It's not necessary to make that change now, though.

Reply:
Done